src/config.rs

```diff
@@ -151,8 +151,17 @@ impl MiniserveConfig {
 
         // Generate some random routes for the favicon and css so that they are very unlikely to conflict with
         // real files.
-        let favicon_route = format!("/{}", nanoid::nanoid!(10, &ROUTE_ALPHABET));
-        let css_route = format!("/{}", nanoid::nanoid!(10, &ROUTE_ALPHABET));
+        let (favicon_route, css_route) = if args.random_route {
+            (
+                format!("/{}", nanoid::nanoid!(10, &ROUTE_ALPHABET)),
+                format!("/{}", nanoid::nanoid!(10, &ROUTE_ALPHABET)),
+            )
+        } else {
+            (
+                format!("{}/{}", route_prefix, nanoid::nanoid!(10, &ROUTE_ALPHABET)),
+                format!("{}/{}", route_prefix, nanoid::nanoid!(10, &ROUTE_ALPHABET)),
+            )
+        };
```

Comment:
Please expand the comment above and describe these two cases and why it's necessary do handle it like this.

Reply:
Done. please check the comment.